.travis.yml

```diff
@@ -5,11 +5,16 @@ dist: xenial
 go:
   - 1.12.x
 
+cache:
+  directories:
+    - $GOPATH/pkg/mod
+
 env:
   global:
     - RUNC_VERSION=v1.0.0-rc8
     - CRIO_VERSION=v1.14.6
     - BUILDAH_VERSION=v1.10.0
+    - GO111MODULE=on
```

Comment:
maybe let's cleanup also "## hack for building on forks" ?

Reply:
It's very handy and I'm using it regularly for building my own fork.

Reply:
What is it for? The travis build seems to proceed successfully without this hack?

Reply:
It's for building in a forked project. Without it it was not possible to build as far as I remember.